interpreter.py

```diff
@@ -118,6 +118,9 @@ def eval_condition(self, predicate, val1, val2):
         else:
             self.exit_not_implemented("Unknown ICmp predicate %d" % predicate)
 
+    def eval_fcondition(self, predicate, val1, val2):
+        self.exit_not_implemented("Unknown ICmp predicate %d" % predicate)
```

Comment:
I wanted to implement it, but then I realized that it is not needed yet. I can remove it.